detection_rules/cli_utils.py

```diff
@@ -132,6 +132,9 @@ def rule_prompt(path=None, rule_type=None, required_only=True, save=True, verbos
 
     for name, options in props.items():
 
+        if name == 'index' and kwargs["type"] == "esql":
```

Comment:
```suggestion
        if name == 'index' and kwargs.get("type") == "esql":
```

Nit should we use .get() here?